src/main/java/PersonalBudget/common/util/Date.java

```diff
@@ -0,0 +1,36 @@
+package PersonalBudget.common.util;
+
+import java.time.LocalDate;
+import java.time.YearMonth;
+
+import static java.time.temporal.TemporalAdjusters.firstDayOfYear;
+import static java.time.temporal.TemporalAdjusters.lastDayOfYear;
+
+public class Date {
+
+    private static final LocalDate localDate = LocalDate.now();
```

Comment:
Please wrap LocalDate.now() in a separate static method and use it in all of the methods in this class- this will be useful for tests